checker/tests/guieffect/TestProgram.java

```diff
@@ -53,11 +53,13 @@ public void doGenericStuff() { // Should be inst. w/ @UI
                     }
                 });
         safecons.runAsync(
+                // :: error: (argument.type.incompatible)
                 new IGenericTask() {
                     @Override
                     public void doGenericStuff() {
                         // Test that the package annotation works
-                        // :: error: (call.invalid.ui)
+                        // Safe here due to anonymous inner class effect inference, but will trigger an error above
+                        // due to safecons.runAsync not taking an @UI IGenericTask
```

Comment:
This is a perfectly reasonable thing to test, but this particular change effectively *removes a test that should still work.*  This test used to check that unannotated methods of a ```@UIPackage``` did in fact have the UI effect.  I'd prefer that you preserve the existence of such a test, though I'm agnostic as to whether that occurs by reverting this test and creating a new one for what this now checks, or by keeping this change and (re)creating a new version of the old test.

Reply:
Sure, my bad! The best way to test that now, is to copy these two calls outside of any anonymous inner class (or into one where the FI is not `@PolyUIType` with an `@PolyUIEffect` method), so that the inference becomes irrelevant to the enclosing method. I'll do the "fully outside of any class option", unless you prefer that I create a subinterface of `IGenericTask` that is forced to be safe? :)